### DISCUSSION TITLE
Download possibilities and download progress bar

### DISCUSSION THREAD

**Discussion by Wlassko** [FreeTubeApp/FreeTube#4732] [Q&A]
Hi, I have version 0.19.2 beta and I have only 2 questions - is planned visible download status bar or something like that? I dont see any progress bar when I try to download video/audio, and last one question is - is downloading working? I tried but nothing downloaded.

Thanks for answers. Have a nice day 🙂 

PS: Sorry if I asking for something what is somewehere here around explained, I dont found it.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> is planned visible download status bar or something like that? I dont see any progress bar when I try to download video/audio

You are right there isnt any progress bar and implementing one is a low priority. There are allot of high priority issues we have to implement first before we get to this. There is an issue open for this though https://github.com/FreeTubeApp/FreeTube/issues/1772

> is downloading working? I tried but nothing downloaded.

Downloading is working and you will get notified when the download started and when its done.


https://github.com/FreeTubeApp/FreeTube/assets/73130443/08562da5-3a78-473b-b8f0-78f40a645317

**Comment by Wlassko** [comment]
Thank you. Have a nice day and thanks for good work 👍
